internal/connection/orchestrator.go

```diff
@@ -83,26 +100,34 @@ func (o *Orchestrator) Connect() {
 	}
 }
 
-func downloadBlob(o *Orchestrator, connectionString string) string {
-	blob, err := o.executor.ExecuteSilent("curl -s --retry 3 " + connectionString)
-
+func downloadBlob(connectionString string) (string, error) {
+	res, err := req.Get(connectionString)
 	if err != nil {
-		panic(err)
+		return "", errors.New(fmt.Sprint("HTTP request to download connection string content failed."))
```

Comment:
shouldn't we include the error in the message here as well? err.Error() gives back a string ;)